tap-gui/configurator/external-plugins.hbs.md

```diff
@@ -11,4 +11,597 @@ own private registry or a plug-in registry if you intend to use a third-party or
+  copying       org.yaml ✔
+  copying       entities.yaml ✔
+  copying       template.yaml ✔
+  copying       catalog-info.yaml ✔
+  copying       index.js ✔
+  copying       package.json ✔
+  copying       README.md ✔
+  templating    .eslintrc.js.hbs ✔
+  copying       Dockerfile ✔
+  copying       README.md ✔
+  templating    package.json.hbs ✔
+  copying       index.test.ts ✔
+  copying       index.ts ✔
+  copying       types.ts ✔
+  copying       app.ts ✔
+  copying       auth.ts ✔
+  copying       catalog.ts ✔
+  copying       proxy.ts ✔
+  copying       scaffolder.ts ✔
+  templating    search.ts.hbs ✔
+  copying       techdocs.ts ✔
+  copying       .eslintignore ✔
+  templating    .eslintrc.js.hbs ✔
+  copying       cypress.json ✔
+  templating    package.json.hbs ✔
+  copying       android-chrome-192x192.png ✔
+  copying       apple-touch-icon.png ✔
+  copying       favicon-16x16.png ✔
+  copying       favicon-32x32.png ✔
+  copying       favicon.ico ✔
+  copying       index.html ✔
+  copying       manifest.json ✔
+  copying       robots.txt ✔
+  copying       safari-pinned-tab.svg ✔
+  copying       .eslintrc.json ✔
+  copying       app.js ✔
+  copying       App.test.tsx ✔
+  copying       App.tsx ✔
+  copying       apis.ts ✔
+  copying       index.tsx ✔
+  copying       setupTests.ts ✔
+  copying       LogoFull.tsx ✔
+  copying       LogoIcon.tsx ✔
+  copying       Root.tsx ✔
+  copying       index.ts ✔
+  copying       SearchPage.tsx ✔
+  copying       EntityPage.tsx ✔
+
+ Moving to final location:
+  moving        plugin-wrappers ✔
+  init          git repository ◜
+🥇  Successfully created plugin-wrappers
+
+
+ All set! Now you might want to:
+  Run the app: cd plugin-wrappers && yarn dev
+  Set up the software catalog: https://backstage.io/docs/features/software-catalog/configuration
+  Add authentication: https://backstage.io/docs/auth/
+```
+
+This command scaffolds a backstage project structure.
+
+We'll start by putting into place our `.yarnrc` which will tell yarn to point to
+our verdaccio server.
+
+```shell
+$ cd plugin-wrappers
+$ echo 'registry "http://localhost:4873"' > .yarnrc
+```
+
+Before installing our dependencies we will remove the packages directory, which
+contains a scaffolded backstage `app` and `backend`. This is because we are only
+making a plugin repository and we don't need to download the dependencies of
+`app` and `backend`.
+
+```shell
+$ rm -rf packages/
+```
+
+And you need to remove the packages directory from the yarn workspaces.
+```diff
+diff --git a/package.json b/package.json
+index 00d64c9..77f38f3 100644
+--- a/package.json
++++ b/package.json
+@@ -24,7 +24,6 @@
+   },
+   "workspaces": {
+     "packages": [
+-      "packages/*",
+       "plugins/*"
+     ]
+   },
+```
+
+```shell
+$ yarn install
+```
+
+Which will install the backstage cli and a few other dependencies.
+
+### tech-insights frontend wrapper
+
+Next you can use the backstage cli to create your plugin wrapper for `tech-insights`
```

Comment:
```suggestion
Next you can use the backstage cli to create your plugin wrapper for TechInsights
```

Reply:
I'd prefer not to add this suggestion. The reason I style it as `tech-insights` in this instance is because that more closely reflects the [backstage plugin name](https://github.com/backstage/backstage/blob/master/plugins/tech-insights/package.json#L2) and the [directory](https://github.com/backstage/backstage/tree/master/plugins/tech-insights) that plugin is in.

If you still think it's worthwhile to style it more like the README prose (as opposed to using a styling more similar to a technical name), then I think we go with

```suggestion
Next you can use the backstage cli to create your plugin wrapper for Tech Insights
```